.github/workflows/static-checks.yaml

```diff
@@ -43,7 +43,7 @@ jobs:
     - name: Install dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends build-essential haveged
+        sudo apt-get install -y --no-install-recommends build-essential haveged jq
```

Comment:
This is not needed, as nydus are not running as part of the `static-checks.yaml`.
Please, drop it, as a whole lot of rework of the static checks has already been done as part of https://github.com/kata-containers/kata-containers/pull/7976

Reply:
Done.